ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -615,26 +651,32 @@
       (for [id service-ids]
         (update-date-hash-with-null-route-hash-id db (:id id))))))
 
-(defn- call-generate-date-hash [db packages]
-  (let [package-count (count packages)]
+(defn- call-generate-date-hash [db packages user future]
+  (let [package-count (count packages)
+        recalculation-id (when packages
+                           (:gtfs/recalculation-id (start-hash-recalculation db package-count user)))]
   (doall
```

Comment:
Old code, but isn't `doall` redundant here because of `dotimes`?

Reply:
yes. Removed.